src/cascadia/TerminalApp/TerminalWindow.cpp

```diff
@@ -900,6 +900,12 @@ namespace winrt::TerminalApp::implementation
                     if (!focusedObject)
                     {
                         focusedObject = winrt::Windows::UI::Xaml::Media::VisualTreeHelper::GetParent(focusedElement);
+

```

Comment:
looks like someone snuck in this branch 😉

Reply:
Yea. I think it came from run code format.

Reply:
Oh.. nvm. doh